x/ethbridge/types/msg_burn.go

```diff
@@ -0,0 +1,64 @@
+package types
+
+import (
+	"encoding/json"
+
+	gethCommon "github.com/ethereum/go-ethereum/common"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// MsgBurnEth defines a message for burning ethereum and triggering a related event
+type MsgBurn struct {
```

Comment:
This and the other `ethbridge` msgs can be condensed in a single `msgs.go` file. Non-blocker though